CRAN_Release.cmd

```diff
@@ -359,7 +359,7 @@ R
 .libPaths()   # should be just 2 items: revdeplib and the base R package library
 update.packages(ask=FALSE)
 # if package not found on mirror, try manually a different one:
-install.packages("MIAmaxent", repos="http://cloud.r-project.org/")
+install.packages("<pkg>", repos="http://cloud.r-project.org/")
```

Comment:
@mattdowle is this intentional?

Reply:
Yes. MIAmaxent was just an old example. This .cmd file is semi-automatic: I press F5 on each line to send to the currently active R console.